source/ranges.tex

```diff
@@ -16995,6 +16995,9 @@
     template<class R2, class V2, class Alloc2, class Unused>
       requires @\libconcept{same_as}@<typename generator<R2, V2, Alloc2>::yielded, yielded>
         auto yield_value(ranges::elements_of<generator<R2, V2, Alloc2>&&, Unused> g) noexcept;
+    template<class R2, class V2, class Alloc2, class Unused>
+      requires same_as<typename generator<R2, V2, Alloc2>::yielded, yielded>
```

Comment:
Do we want to use `\libconcept{same_as}` or was it intentionally omitted because it's on the declaration immediately above?

Reply:
We want \libconcept everywhere a library concept appears. This is refactoring-safe and if we ever want to have color highlighting for concept names (for example), we actually can do that.

Reply:
```suggestion
      requires @\libconcept{same_as}@<typename generator<R2, V2, Alloc2>::yielded, yielded>
```

Reply:
Thanks, fixed in two places!